XIVLauncher/App.config

```diff
@@ -20,6 +20,9 @@
             <setting name="Language" serializeAs="String">
                 <value>1</value>
             </setting>
+            <setting name="LauncherLanguage" serializeAs="String">

```

Comment:
We aren't using these settings anymore - you don't have to add them when added to LauncherSettingsV3. They're just here for compatibility.